spring-jdbc/src/main/java/org/springframework/jdbc/object/SqlFunction.java

```diff
@@ -65,7 +65,6 @@ public class SqlFunction<T> extends MappingSqlQuery<T> {
 	 * @see #compile
 	 */
 	public SqlFunction() {
-		setRowsExpected(1);
 	}
```

Comment:
Please revert all changes to `SqlFunction`.

We will remove these invocations when we remove that setter method in conjunction with #34530.